<documentation title="Compound Namespace Depth">
    <standard>
    <![CDATA[
    Compound namespaces with a depth of more than two MUST NOT be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Max depth of 2.">
        <![CDATA[
use ps_metrics_module_v4_0_5\Vendor\Package\SomeNamespace\{
    ps_metrics_module_v4_0_5\SubnamespaceOne\ClassA,
    ps_metrics_module_v4_0_5\SubnamespaceOne\ClassB,
    ps_metrics_module_v4_0_5\SubnamespaceTwo\ClassY,
    ClassZ,
};
        ]]>
        </code>
        <code title="Invalid: Max depth of 3.">
        <![CDATA[
use ps_metrics_module_v4_0_5\Vendor\Package\SomeNamespace\{
    <em>ps_metrics_module_v4_0_5\SubnamespaceOne\AnotherNamespace\ClassA,</em>
    ps_metrics_module_v4_0_5\SubnamespaceOne\ClassB,
    ClassZ,
};
        ]]>
        </code>
    </code_comparison>
</documentation>
